classes/content/sqlicontent.php

```diff
@@ -62,7 +62,7 @@ class SQLIContent
 
     /**
      * Fieldset Holder
-     * @var SQLIFieldsetHolder
+     * @var SQLILocationSet|SQLILocation[]
```

Comment:
You were right, my initial change was wrong - still, I changed the PHPDoc to SQLI**Content**FieldsetHolder , because the class SQLIFieldsetHolder doesn't seem to exist.